File: AndroidStudio/SprintModulo4/app/src/main/java/com/example/sprintmodulo4/MainActivity.java
package com.example.sprintmodulo4;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;


public class MainActivity extends AppCompatActivity {

    private ImageView imageViewProfile;
    private Button buttonContact;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        imageViewProfile = findViewById(R.id.imageViewProfile);
        buttonContact = findViewById(R.id.buttonContact);


        buttonContact.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(MainActivity.this, ContactActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: AndroidStudio/Individual7M5/app/src/main/java/com/example/individual7m5/MainActivity.kt
package com.example.individual7m5

import android.os.Bundle
import android.widget.*
import androidx.appcompat.app.AppCompatActivity
import androidx.core.content.ContextCompat

class MainActivity : AppCompatActivity() {

    private var saldo: Double = 1000.0 // Saldo inicial

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        val radioGroup: RadioGroup = findViewById(R.id.radioGroup)
        val buttonOK: Button = findViewById(R.id.btnOK)

        buttonOK.setOnClickListener {

            when (radioGroup.checkedRadioButtonId) {
                R.id.rbVerSaldo -> verSaldo()
                R.id.rbIngresar -> ingresarDinero()
                R.id.rbRetirar -> retirarDinero()
                R.id.rbSalir -> salir()
                else -> Toast.makeText(this, "Seleccione una opción", Toast.LENGTH_SHORT).show()
            }
        }
    }


    private fun verSaldo() {
        Toast.makeText(this, "Su saldo es: $$saldo", Toast.LENGTH_LONG).show()
    }


    private fun ingresarDinero() {
        val montoIngresar = 100.0
        saldo += montoIngresar
        Toast.makeText(this, "Has ingresado $$montoIngresar. Saldo actualizado: $$saldo", Toast.LENGTH_LONG).show()
    }

    private fun retirarDinero() {
        val montoRetirar = 50.0
        if (montoRetirar > saldo) {
            Toast.makeText(this, "No puedes retirar más dinero del saldo disponible.", Toast.LENGTH_LONG).show()
        } else {
            saldo -= montoRetirar
            Toast.makeText(this, "Has retirado $$montoRetirar. Saldo restante: $$saldo", Toast.LENGTH_LONG).show()
        }
    }

    // Función para salir de la app
    private fun salir() {
        finishAffinity()
    }
}